app/controllers/inventories_controller.rb

```diff
@@ -2,7 +2,7 @@ class InventoriesController < ApplicationController
   before_action :set_inventory, only: [:edit, :update, :destroy]
 
   def index
-    @inventories = Inventory.all
+    @inventories = Inventory.search(search_params).order(:id).page(params[:page])
```

Comment:
```suggestion
    @inventories = Inventory.includes(:model).search(search_params).order(:id).page(params[:page])
```
Prevent N+1 Query: https://pawelurbanek.com/rails-n-1-queries. 
Before: 
![image](https://user-images.githubusercontent.com/48946963/125727779-4298594a-88e4-406c-9dd3-54fc58d8ff72.png)

after: 
![image](https://user-images.githubusercontent.com/48946963/125727794-5c6eaaf9-5e3b-4983-8c51-f43278520d85.png)

Reply:
Oh cool 🙆‍♂️

Reply:
Did the line `belongs_to -> {includes :brand}` in models/inventory.rb give the same effect here as Inventory.includes(:model)? I've doubted about the purpose of that line of code because ror-guilds suggested but actually I don't know the effect.